lib/google/ads/google_ads/version.rb

```diff
@@ -19,7 +19,7 @@
 module Google
   module Ads
     module GoogleAds
-      CLIENT_LIB_VERSION = '8.1.0'.freeze
+      CLIENT_LIB_VERSION = '8.2.0'.freeze
```

Comment:
Make it 9.0.0 since we're removing v3 support.

Reply:
Done.